pages/firststeps.md

```diff
@@ -25,37 +25,37 @@ A very large part of these steps is finding problems with these steps and this M
 There is no official deadline to complete these steps, but most candidates who are approved for the internship program finished the steps within 7-8 days. Good luck!
 
 
-##Step #0: Introductory Steps:
```

Comment:
> #0

Same here, the hashtag here there for a reason. It is a sign for a numbering. I prefer hashtag because it shows how to read it. For example

> Step #0: Introductory Steps:

you can read that as

> Step Number 0: Introductory Steps

Please wait for other contributors for more comprehensive review.